decidim-core/lib/decidim/participatory_space_resourceable.rb

```diff
@@ -100,6 +100,17 @@ def resource_description
       def visible?
         published? && !try(:private_space?)
       end
+
+      # Defines a way to get the user roles for the current participatory space.
+      # You should overwrite this method in the implementer class to define how
+      # to get the correct values.
+      #
+      # role_name - A symbol or string identifying the role name
+      #
+      # Returns an ActiveRecord::Relation.
```

Comment:
Correct, she should only export her assigned proposals. I haven't been able to check *what* is exported because sending mail doesn't work in staging, but I assume it is like this.